src/ontology/reports/omim.subclass.added.robot.tsv

```diff
@@ -0,0 +1,1241 @@
+subject_mondo_id	subject_mondo_label	object_mondo_id	subject_source_id	object_source_id	object_mondo_label
+ID	LABEL	SC %			
+MONDO:0009335	hemolytic uremic syndrome, atypical, susceptibility to, 1	MONDO:0957097	OMIM:235400	OMIMPS:235400	hereditary hemolytic uremic syndrome
+MONDO:0013041	atypical hemolytic-uremic syndrome with I factor anomaly	MONDO:0957097	OMIM:612923	OMIMPS:235400	hereditary hemolytic uremic syndrome
+MONDO:0013044	atypical hemolytic-uremic syndrome with thrombomodulin anomaly	MONDO:0957097	OMIM:612926	OMIMPS:235400	hereditary hemolytic uremic syndrome
+MONDO:0014005	immunoglobulin-mediated membranoproliferative glomerulonephritis	MONDO:0957097	OMIM:615008	OMIMPS:235400	hereditary hemolytic uremic syndrome
+MONDO:0013040	atypical hemolytic-uremic syndrome with MCP/CD46 anomaly	MONDO:0957097	OMIM:612922	OMIMPS:235400	hereditary hemolytic uremic syndrome
+MONDO:0013043	atypical hemolytic-uremic syndrome with C3 anomaly	MONDO:0957097	OMIM:612925	OMIMPS:235400	hereditary hemolytic uremic syndrome
+MONDO:0013042	atypical hemolytic-uremic syndrome with B factor anomaly	MONDO:0957097	OMIM:612924	OMIMPS:235400	hereditary hemolytic uremic syndrome
+MONDO:0012775	thrombocytopenia 4	MONDO:0100241	OMIM:612004	OMIMPS:313900	inherited thrombocytopenia
```

Comment:
Is this case 2 or case 3 in the synchronisation table?

Reply:
The `subclass.added` files are for case 3 ([google doc](https://docs.google.com/document/d/1H8fJKiKD-L1tfS-2LJu8t0_2YXJ1PQJ_7Zkoj7Vf7xA/edit?disco=AAAA_ExT1F0)):
_Case 3: SCR is direct in the source, but not at all in Mondo_

Reply:
Good, thanks!